tendrl/integrations/gluster/check_cluster_status.py

```diff
@@ -0,0 +1,16 @@
+import etcd
+
+
+def run():
+    try:
+        NS._int.client.write(
+            "/clusters/{0}/GlobalDetails/status".format(
+                NS.tendrl_context.integration_id
```

Comment:
This is monitor node, I guess it wont have integration id. I feel you have to loop through all the clusters and set the status

Reply:
Ack. Will do that.